include/intx/int128.hpp

```diff
@@ -63,46 +63,56 @@ struct uint<128>
 {
     static constexpr unsigned num_bits = 128;
 
-    uint64_t lo = 0;
-    uint64_t hi = 0;
+    uint64_t words[2]{};
```

Comment:
Should comment that this is in "little endian" order, i.e. words[0] is lo.

Reply:
This is little endian order.

Reply:
Yeah I wrote that first, was confused at some point.